### PR TITLE
Fixed the bug related to mime_type on ajax request

### PR DESCRIPTION
**Pull request by larodiel**
- Added the languages pt_BR and es_ES
- Fixed the bug where the library on AJAX requests was broking because it was expecting a string and the server was returning an array

### CODE REVIEW

File: public/class-media-search-enhanced.php [hunk 1]
```diff
@@ -161,9 +161,15 @@ public static function posts_clauses( $pieces ) {
...
-				$pieces['where'] .= $wpdb->prepare( " AND $wpdb->posts.post_mime_type LIKE %s", $like );
+					$mime_types = is_array( $vars['post_mime_type'] ) ? $vars['post_mime_type'] : [ $vars['post_mime_type'] ];
+					$like_clauses = array();
+
+					foreach( $mime_types as $mime_type ) {
+							$mime_type_like = '%' . $wpdb->esc_like( $mime_type ) . '%';
+							$like_clauses[] = "$wpdb->posts.post_mime_type LIKE '$mime_type_like'";
+					}
+
+					$pieces['where'] .= " AND (" . implode( ' OR ', $like_clauses ) . ")";
```
Comment (yoren) [inline public/class-media-search-enhanced.php]: Are the indents here a bit off?

Also, mind if changing `$like_clauses[] = "$wpdb->posts.post_mime_type LIKE '$mime_type_like'";` to use `$wpdb->prepare()`?

**Review by yoren — COMMENTED** [review]
Thanks @larodiel - please find my feedback from the comment.

File: public/class-media-search-enhanced.php (hunk 1 above)
Comment (larodiel) [reply public/class-media-search-enhanced.php]: Hi @yoren 
prepare has been added and indentation fixed.
Would be great if we had a `.editorconfig` or prettier file to keep the pattern `.prettierrc`

Comment (yoren) [reply public/class-media-search-enhanced.php]: Thank you. I'll do some test and merge if no further issues + do the release to wp.org over the weekend.

**Review by yoren — APPROVED** [review]
@larodiel Thanks! Can you give me an example that when will the `post_mime_type` be an array? The default Media Library doesn't seem to have an UI to make it so. I wanted to make it clear in the change log or also add some information in README.

**Comment by larodiel** [comment]
> @larodiel Thanks! Can you give me an example that when will the `post_mime_type` be an array? The default Media Library doesn't seem to have an UI to make it so. I wanted to make it clear in the change log or also add some information in README.

@yoren When the user add an image using the Gutenberg block `image` and they try to search for an image on the library, the ajax request was returning error 500, because on this section, the mime type is requested using array instead of string

**Comment by yoren** [comment]
@larodiel Thanks - that information was very valuable! If you have a wp.org account, feel free to add yourself to the "Contributors" list in README. I'll release 0.8.2 soon

**Review by yoren — APPROVED** [review]

**Comment by yoren** [comment]
Hey @larodiel - It's been a while but I finally published the new release today. Thank you!

**Pull request merged**